less/admin/AdminNav.less

```diff
@@ -180,6 +188,10 @@
 
 }
 
+.AdminLinkButton-description {
+  white-space: normal;
```

Comment:
I think adding 5px of left padding would be nice as well, to make the description symmetrical with the title.